src/include/processor/operator/call.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "main/db_config.h"
+#include "processor/operator/physical_operator.h"
+
+namespace kuzu {
+namespace processor {
+
+struct CallLocalState {
+    main::ConfigurationOption option;
+    std::shared_ptr<binder::Expression> optionValue;
+    bool hasExecuted = false;
```

Comment:
this should be a sharedState because multiple thread will execute call repeatedly.

Or you need force the pipeline to be single threaded and then you can make it a local state

Reply:
set it single threaded